playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/containerized_upgrade.yml

```diff
@@ -0,0 +1,9 @@
+- name: Update system_units
+  script: ../files/ensure_system_units_have_version.sh {{ openshift.common.service_type }} {{ openshift.common.deployment_type }} {{ g_new_version }}
+
+- name: Verifying the correct version was configured
+  shell: grep {{ verify_upgrade_version }} {{ item }}
```

Comment:
I'm not sure about faster, but it is safer, since there is no chance of running extraneous commands through the shell.